### ISSUE TITLE
Open file manager keybind no longer works

### ISSUE THREAD

**Issue by Stoppedpuma** [woelper/oculante#442]

**Comment by woelper** [comment]
This still works on my machine (tm) - is this on Linux? Is the keybind correct?

**Comment by Stoppedpuma** [comment]
Yes this is on linux, I have the keybind set to "o" on my system and it doesn't seem to open. Probably should've mentioned this in the issue description but I do not have the OS native file handler in the build options so this issue is for the Oculante file manager.

Here's a video showing this (You can see key presses in the bottom right):

[open.webm](https://github.com/user-attachments/assets/ab632bd5-722f-4ce0-927f-9b932afb07a8)
